src/modules/homepage/IncomeExpense.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Stack } from '@mui/material';
+import { ReactNode } from 'react';
+import { theme } from 'src/styles/theme';
+import { colors } from 'src/styles/colors';
+import CameraAltIcon from '@mui/icons-material/CameraAlt';
+import { Regular3 } from '../../shared-components/Typography/Regular3';
+
+interface IncomeExpenseCardProps {
+  children: ReactNode;
+  bgcolor: string;
+  color: string;
+  content: string;
+  amount: string;
+}
+
+export const IncomeExpenseCard: React.FC<IncomeExpenseCardProps> = ({
+  children,
+  bgcolor,
+  color,
+  content,
+  amount,
+}) => {
+  return (
+    <Box
+      borderRadius={theme.shape.borderRadius * 7}
+      bgcolor={bgcolor}
```

Comment:
We know that the card can be used for incomes or for expenses. Instead of passing color and bgcolor, pass a prop that specify if the component is used for incomes or expenses. Something like this:
```
enum TransactionType {
  income = 'Income',
  expense = 'Expense'
};

```

You can place this enum in the `types` folder and export it. Then add `transactionType` prop to `IncomeExpenseCard` and depending on the value set bg color, icon, color and title text.

Reply:
I used props named transactionType.